pytest-embedded-idf/pytest_embedded_idf/serial.py

```diff
@@ -161,6 +161,8 @@ def flash(self, app: Optional[IdfApp] = None) -> None:
                 else:
                     _args.append(str(v))
 
+        _args.extend(['--baud', '1000000'])
```

Comment:
let's keep the behavior the same as `idf.py flash`

```
❯ idf.py flash --help
Usage: idf.py flash [OPTIONS]

  Flash the project.

Options:
  -C, --project-dir PATH         Project directory.
  -D, --define-cache-entry TEXT  Create a cmake cache entry. This option can be used at most once either globally, or for one subcommand.
  -b, --baud INTEGER             Baud rate for flashing. It can imply monitor baud rate as well if it hasn't been defined locally. The
                                 default value can be set with the ESPBAUD environment variable. This option can be used at most once
                                 either globally, or for one subcommand.
```

i.e., if ESPBAUD set, use the set value. if not, use the suggested value 1000000

Reply:
Agree